### PR TITLE
Bug 1765301: UPSTREAM: 89379: deflake grace period for pod deletion in node e2e

### PR DESCRIPTION
**Pull request by giuseppe**
Fixes: rhbz#1765301

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by giuseppe** [comment]
@derekwaynecarr @mrunalp PTAL

**Comment by haircommander** [comment]
/retitle Bug 1765301: UPSTREAM: 89379: deflake grace period for pod deletion in node e2e

File: vendor/k8s.io/kubernetes/test/e2e/node/pods.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ var _ = SIGDescribe("Pods Extended", func() {
 		/*
 			Release : v1.15
 			Testname: Pods, delete grace period
-			Description: Create a pod, make sure it is running. Using the http client send a 'delete' with gracePeriodSeconds=30. Pod SHOULD get deleted within 30 seconds.
+			Description: Create a pod, make sure it is running. Using the http client send a 'delete' with gracePeriodSeconds=30. Pod SHOULD get terminated within gracePeriodSeconds and removed from API server within a window.
 		*/
 		ginkgo.It("should be submitted and removed [Flaky]", func() {
```
Comment (sjenning) [inline vendor/k8s.io/kubernetes/test/e2e/node/pods.go:59]: in the upstream PR the `[Flaky]` is dropped

File: HACKING.md [hunk 2]
```diff
@@ -286,7 +286,7 @@ For convenience, you can use `hack/cherry-pick.sh` to generate patches for
 Origin from upstream commits.
 
 The purpose of this command is to allow you to pull individual commits from a
-local kubernetes repository into origin's vendored kuberenetes in a fully
+local kubernetes repository into origin's vendored kubernetes in a fully
```
Comment (sjenning) [inline HACKING.md:289]: this change is not in the upstream PR

File: vendor/k8s.io/kubernetes/test/e2e/node/pods.go (hunk 1 above)
Comment (giuseppe) [reply vendor/k8s.io/kubernetes/test/e2e/node/pods.go]: thanks for the review.  Restored now.  I've added a new commit to regenerate the bindata so tests don't fail

File: HACKING.md (hunk 2 above)
Comment (giuseppe) [reply HACKING.md]: dropped

**Comment by giuseppe** [comment]
@sjenning tests are passing now

**Pull request merged**